content/6-development/6-1-contribution-guidelines.md

```diff
@@ -112,12 +112,12 @@ SecRule ARGS "foo" "id:1,phase:1,pass,t:none"
 CRS uses `\x5c` to represent the backslash `\` character in regular expressions. Some of the reasons for this are:
 
 * It's portable across web servers and WAF engines: it works with Apache, Nginx, and Coraza.
-* It works with the [crs-toolchain]({{% ref "crs_toolchain" %}}) for building optimized regular expressions.
+* It works with the [crs-toolchain]({{< ref "6-development/6-2-crs-toolchain.md" >}}) for building optimized regular expressions.
```

Comment:
Can we remove `6-development/` as we are already in "folder" `6-development`?
There are more of those links in this file here and in 6-2, 6-3 and 6-6.
I'm not entirely sure. Just asking questions ;-)

Reply:
I see. We should use one or the other syntax here.

Reply:
How about putting some guidelines for the use of syntax in README?

Reply:
fixed all in [d14fc43](https://github.com/coreruleset/documentation/pull/137/commits/d14fc4333e2a060f5c0ee418c65e7e1e97c8d3e4) and add doc in [4828b8d](https://github.com/coreruleset/documentation/pull/137/commits/4828b8d882017e5cb0ef279eed6ddd2f536477ab).